Kconfig

```
@@ -1473,6 +1473,16 @@ menu "LVGL configuration"
 			int "Font manager name max length"
 			depends on LV_USE_FONT_MANAGER
 			default 32
+
+		config LVGL_VERSION_MAJOR
+			int "Major version"
```

Can someone change it by accident? If so I suggest 

```suggestion
			int "Major version (used internally to check the version, do not change it)"
```

---

Added.